src/test/java/org/broadinstitute/hellbender/utils/runtime/StreamingProcessControllerUnitTest.java

```diff
@@ -23,163 +19,75 @@
 //
 public class StreamingProcessControllerUnitTest extends BaseTest {
     private final static String NL = System.lineSeparator();
-    private final static String PYTHON_PROMPT = ">>> ";
 
     @Test(groups = "python", timeOut = 10000)
-    public void testSerialCommands() throws TimeoutException, IOException {
-        // start an interactive Python session with unbuffered IO
-        final ProcessSettings processSettings = new ProcessSettings(new String[] {"python", "-i", "-u"});
-
-        final StreamingProcessController controller = new StreamingProcessController(processSettings, PYTHON_PROMPT, true);
-        Assert.assertTrue(controller.start());
-
-        // consume the Python startup banner debris, but don't try validate it
-        controller.getProcessOutputByPrompt();
+    public void testSerialCommands() throws IOException {
```

Comment:
Just for completeness, it might be good to have a few more test cases here.  You could just make a data provider with 5 or so simple chained expressions.

Reply:
Sure, done.